src/lib/atoms/derived/index.ts

```
@@ -0,0 +1,130 @@
+
+// User's active conversations
+export const activeConversationsAtom = atom(get => {
+  const conversations = get(conversationsAtom)
+  const user = get(userAtom)
+  
+  if (!user) return []
+  
+  // ConversationWithUser doesn't have participant1_id/participant2_id
+  // It has sender and recipient properties
+  return conversations.filter(conv => {
+    const userId = typeof user === 'object' && 'id' in user ? (user as any).id : null
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)